perfTestUtils/dataStructures.go

```diff
@@ -182,17 +182,25 @@ type PerfStats struct {
 	PeakMemory           uint64
 	ServiceResponseTimes map[string]int64
 	ServiceTransCount    map[string]*uint64
+	ServiceErrorCount    map[string]*uint64
 	ServiceTPS           map[string]float64
 	OverAllTransCount    uint64
+	OverAllErrorCount    uint64
 	OverAllTPS           float64
 	MemoryAudit          []uint64
 	TestPartitions       []TestPartition
```

Comment:
Added TestTimeEnd for reporting. Renameve TestDate to TestTimeStart for consistency.